src/netstandard/WampSharp/WAMP2/V2/Api/CalleeProxy/ObservableCalleeProxyInterceptor.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Reactive.Linq;
+using System.Reflection;
+
+namespace WampSharp.V2.CalleeProxy
+{
+    internal class ObservableCalleeProxyInterceptor<T> : CalleeProxyInterceptorBase<T>
+    {
+        public ObservableCalleeProxyInterceptor(MethodInfo method, IWampCalleeProxyInvocationHandler handler, ICalleeProxyInterceptor interceptor) : base(method, handler, interceptor)
+        {
+        }
+
+        private class ObsProgress : IProgress<T>
```

Comment:
There exists an Rx method for [this](https://github.com/dotnet/reactive/blob/346807e8726e092a1d042a9dcb7d04e00f088c89/Rx.NET/Source/src/System.Reactive/Observer.Extensions.cs#L340).

Reply:
Nice! I didn't know, thnx.